### ISSUE TITLE
ElementaryOS

### ISSUE THREAD

**Issue by aliceemanueli** [babluboy/bookworm#370]
Good morning!  This app is not available in the ElementaryOS App Store... Please bring it to ElementaryOS.  Why do we miss it!

**Comment by babluboy** [comment]
Yes, with the last change in AppCenter to use Flatpak for Apps, Bookworm is not updated and re-submitted to AppCenter.

Bookworm was made for Elementary OS so as soon as I have some time to install the latest version of eOS, I will make the necessary changes and resubmit Bookworm to AppCenter.


Will keep this issue open to remind me to work on Bookworm resubmission to AppCenter
